<!--Basic Launch file incuding node & arguments components-->
<launch>
    <arg name="enabled" default="true"/>
    <!-- ARDRONE NODE -->
    <include file="$(find landing_pkg)/launch/ardrone.launch"/>
    <!--or with ardrone simulator in gazebo-->
    <!-- <include file="$(find cvg_sim_gazebo)/launch/ardrone_testworld.launch"/> -->

    <!-- ARSYS NODE -->
    <include file="$(find landing_pkg)/launch/arsys.launch"/>
    <!--<include file="$(find ar_sys)/launch/single_board.launch"/>-->

    <!-- TRANSFORMATIONS NODES (From PoseStamped to x, y, z float -->
    <include file="$(find landing_pkg)/launch/transformations.launch"/>

    <!-- PID X, Y, Z -->
    <include file="$(find landing_pkg)/launch/position_pids.launch"/>    

    <node name="vel_controller" type="vel_controller" pkg="landing_pkg" output="screen">
        <param name="enabled" value="$(arg enabled)"/>
        <param name="has_to_reach_quota" value="true"/>
        <param name="landing_quota" value="2000"/>
        <param name="zeta_land_vel" value="0.2"/>
        <param name="quota_reached_toleration" value="250"/>
    </node>

    <arg name="video_rectified" default="false" />

    <node name="quota_monitor" type="quota_monitor" pkg="landing_pkg" output="screen"/>

    <!--<node name="rel_graph" type="rqt_graph" pkg="rqt_graph"/>-->

</launch>